sign-plugin-docs: information of usage of a new token has been added to the docs (#72912)
grafana/grafana: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md

If you intend to only use the plugin within your organization, sign it under a _private_ [signature level](#plugin-signature-levels).

## Generate a token

> To verify ownership of your plugin, generate access token that you'll use every time you need to sign a new version of your plugin.

Reviewer: `generate access token` -> `generate an access token`

1. [Create a Grafana Cloud account](/signup).

> 1. Login into your account and navigate to "My Account" -> Security-> Access Policies. And click "Create access pilicy"

Reviewer 2: ```suggestion
1. Login into your account and navigate to **My Account > Security > Access Policies**. Click **Create access policy**.
```

Author: Done! Thank you!

>    Realm: has to be your-org-name (all-stacks)\

Reviewer 2: ```suggestion
   Realm: has to be your-org-name (all-stacks)
   
```

Author: Done! Thank you!

   Scope: plugins:write

   {{< figure src="/static/img/docs/plugins/create-access-policy.png" class="docs-image--no-shadow" max-width="650px" >}}

> 1. Once policy created, create a new tocker by clicking "Create token" button.\

Reviewer 2: ```suggestion
1. Click **Create token** to create a new token.
```

Author: Done! Thank you!

>    Token name could be anything\
>    "Expration date" is oprional although we stringly advise to change tokens from time to time for security reason.\
> 
>    {{< figure src="/static/img/docs/plugins/create-access-policy-token.png" class="docs-image--no-shadow" max-width="650px" >}}
> 
>    Click "Create" and Copy the token

Reviewer 2: ```suggestion

   The expiration date field is optional, though you should change tokens periodically for increased security.

   {{< figure src="/static/img/docs/plugins/create-access-policy-token.png" class="docs-image--no-shadow" max-width="650px" >}}

1. Click **Create** and save a copy of the token somewhere secure for future reference. 
```

Author: Done! Thank you!
